source/routing/tests/next_stop_time_test.cpp

```diff
@@ -2047,7 +2047,7 @@ struct classic_freq_dataset {
     classic_freq_dataset() {
         vp = type::ValidityPattern(date(2012,7,7), "11");
 
-        vj.validity_pattern = &vp;
+        vj.validity_patterns[type::RTLevel::Theoric] = &vp;
```

Comment:
It's not possible, you can't set with a getter (or rather I won't :wink: ) and I'm not a fan of putting a setter for this.
It's a draw,  @TeXitoi or @xlqian do you have a preference ?